index.bs

```diff
@@ -169,6 +169,12 @@ interface VRDisplay : EventTarget {
    * created without preserveDrawingBuffer set to true will be cleared.
    */
   void submitFrame();
+
+  attribute EventHandler onactivate;
+  attribute EventHandler ondeactivate;
+  attribute EventHandler onblur;
+  attribute EventHandler onfocus;
+  attribute EventHandler onpresentchange;
```

Comment:
It would be a little easier to discuss design in dedicated threads, rather than subthreads of a PR.

What matters here are the use cases and what the typical programming experience would be like. Having said that, exposing state transitions through promises is the latest trend and if that makes sense for your API that'd the best to align on.